src/app/components/searchbar/searchbar.component.ts

```diff
@@ -0,0 +1,26 @@
+import { Component, OnInit, Output, EventEmitter } from '@angular/core';
+
+@Component({
+  selector: 'app-searchbar',
+  templateUrl: './searchbar.component.html',
+  styleUrls: ['./searchbar.component.css']
+})
+export class SearchbarComponent implements OnInit {
+
+  @Output() searched = new EventEmitter();
+
+  constructor() { }
+
+  ngOnInit() {
+  }
+
+  onSearch(searchValue: string = null) {
```

Comment:
No need to initialize the `searchValue` with null. that way you can remove the if-else check